app/src/main/java/ca/marshallasch/veil/database/Database.java

```diff
@@ -125,6 +135,10 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
         if (oldVersion < 4) {
             Migrations.upgradeV4(db);
         }
+
+        if (oldVersion < 6) {
```

Comment:
What happened to v5?

Reply:
I messed up the migration again and had to go to v6 I meant to insert a comment but forgot about it.